### PR TITLE
DEV: removes jQuery usage from extend-for-poll

### PR DESCRIPTION
**Pull request by jjaffeux**

### CODE REVIEW

File: plugins/poll/assets/javascripts/initializers/extend-for-poll.js [hunk 1]
```diff
@@ -81,36 +80,33 @@ function initializePolls(api) {
...
-      const $poll = $(pollElem);
-      const pollName = $poll.data("poll-name");
+    pollNodes.forEach((pollNode) => {
+      const pollName = pollNode.dataset.pollName;
       let poll = polls[pollName];
       let pollPost = post;
       let vote = votes[pollName] || [];
 
-      const quotedId = $poll.parent(".expanded-quote").data("post-id");
+      const quotedId = pollNode.closest(".expanded-quote")?.dataset.postId;
```
Comment (jjaffeux) [inline plugins/poll/assets/javascripts/initializers/extend-for-poll.js:91]: I did few tests and it seems to work as expected

**Comment by CvX** [comment]
Looks like the poll tests fail?

**Comment by jjaffeux** [comment]
> Looks like the poll tests fail?

yes saw that was out for lunch will look after

**Comment by jjaffeux** [comment]
> Looks like the poll tests fail?

Ok I dropped changing setInterval for now and it should pass

**Review by CvX — APPROVED** [review]

**Pull request merged**